client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2430,18 +2433,29 @@ Undocumented.prototype.checkSiteAddressValidation = function( siteId, siteAddres
  *
```

Comment:
We should probably update the description of this method since we can now also have a `.blog` address.